### DISCUSSION TITLE
verification on flathub

### DISCUSSION THREAD

**Discussion by corngoblin** [LizardByte/community#495] [Sunshine] [answered]
I noticed the sunshine flatpak on flathub isn't verified.
Can we get that verified? I'm not real sure if that is just some random flatpak, or real.
I know there is one in the releases page, but it'd be nice to know if the one on flathub is fine.

Thank you for your time.

**Comment by ReenigneArcher** [comment]
Once this is resolved: https://github.com/flathub/dev.lizardbyte.app.Sunshine/issues/36

**Reply by ReenigneArcher** [reply, marked as answer]
Also: https://github.com/flathub/flathub/issues/5663
